<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReferralsWindow</class>
 <widget class="QMainWindow" name="ReferralsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1260</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1260</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1260</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Forsyth Family Practice Center - [ TEMPLATE ]</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/newPrefix/imgs/app-bg.png);

QGroupBox, QFrame {
	border-image: none;
	border: none;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_NavBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>1221</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none; border-image: none;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1201</width>
       <height>42</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="Layout_NavBar">
      <property name="spacing">
       <number>20</number>
      </property>
      <item>
       <widget class="QPushButton" name="Nav_LogoutBtn">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border-image: none;
	background-color: rgb(30, 147, 143);
	color: black;
	border: 2px solid black;
}

QPushButton::hover {
	background-color: rgb(14, 81, 85);
	color: white;
}</string>
        </property>
        <property name="text">
         <string>Logout</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Nav_Appointments">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border-image: none;
	background-color: #6ECCAF;
	color: black;
	border: 2px solid black;
}

QPushButton::hover {
	background-color: rgb(139, 231, 100);
	color: white;
}</string>
        </property>
        <property name="text">
         <string>Appointments</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Nav_CheckinBtn">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border-image: none;
	background-color: #6ECCAF;
	color: black;
	border: 2px solid black;
}

QPushButton::hover {
	background-color: rgb(139, 231, 100);
	color: white;
}

</string>
        </property>
        <property name="text">
         <string>Check-in</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Nav_CheckoutBtn">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border-image: none;
	background-color: #6ECCAF;
	color: black;
	border: 2px solid black;
}

QPushButton::hover {
	background-color: rgb(139, 231, 100);
	color: white;
}</string>
        </property>
        <property name="text">
         <string>Check-out</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Nav_MakeReferralBtn">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border-image: none;
	background-color: #6ECCAF;
	color: black;
	border: 2px solid black;
}

QPushButton::hover {
	background-color: rgb(139, 231, 100);
	color: white;
}
</string>
        </property>
        <property name="text">
         <string>Make Referral</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Nav_LabOrdersBtn">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border-image: none;
	background-color: #6ECCAF;
	color: black;
	border: 2px solid black;
}

QPushButton::hover {
	background-color: rgb(139, 231, 100);
	color: white;
}</string>
        </property>
        <property name="text">
         <string>Lab Orders</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Nav_ApproveAppointmentsBtn">
        <property name="minimumSize">
         <size>
          <width>240</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border-image: none;
	background-color: #6ECCAF;
	color: black;
	border: 2px solid black;
}

QPushButton::hover {
	background-color: rgb(139, 231, 100);
	color: white;
}</string>
        </property>
        <property name="text">
         <string>App. Approve via Portal</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6"/>
   </widget>
   <widget class="QFrame" name="Main_Frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>1221</width>
      <height>559</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	border-image: none;
	background-color: rgb(52, 77, 103);
}


QLineEdit, QComboBox, QDateEdit, QDateTimeEdit {
	border-image: none;
    border: none;
	min-width: 200px;
	height: 30px;
	background-color: #F3ECB0;
	color: #344D67;
	font-family: &quot;MS Shell Dlg 2&quot;;
	font-size: 11;
}

QLineEdit {
	padding-left: 5px;
	padding-right: 5px;
}

QLabel {
	color: white;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="InputFrame">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>1251</width>
       <height>561</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: none;
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>411</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="Layout_PatientFirstName">
         <property name="spacing">
          <number>7</number>
         </property>
         <item>
          <widget class="QLabel" name="Label_PatientFirstName">
           <property name="font">
            <font>
             <family>Lato</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>First Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="LineEdit_PatientFirstName">
           <property name="font">
            <font>
             <family>MS Shell Dlg 2</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Example</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>50</y>
        <width>456</width>
        <height>38</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="Layout_DocOrPrac">
       <item>
        <widget class="QLabel" name="Label_DoctorPractitioner">
         <property name="font">
          <font>
           <family>Lato</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Doctor or Practitioner:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="ComboBox_DoctorPractitioner">
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>First Item</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Second Item</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>110</y>
        <width>456</width>
        <height>38</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="Layout_comboBox_Creation_Date">
       <item>
        <widget class="QLabel" name="Label_CreationDate">
         <property name="font">
          <font>
           <family>Lato</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Creation Date:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="DateEdit_CreationDate">
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>11</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="Btn_CreateReferral">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>320</y>
        <width>411</width>
        <height>61</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Lato</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border-image: none;
	background-color: #6ECCAF;
	color: black;
	border: 2px solid black;
}

QPushButton::hover {
	background-color: rgb(139, 231, 100);
	color: white;
}</string>
      </property>
      <property name="text">
       <string>Create Referral</string>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>411</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="Layout_LastName">
       <property name="spacing">
        <number>7</number>
       </property>
       <item>
        <widget class="QLabel" name="Label_PatientLastName">
         <property name="font">
          <font>
           <family>Lato</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Last Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LineEdit_PatientLastName">
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Example</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>411</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="Layout_DateOfBirth">
       <property name="spacing">
        <number>7</number>
       </property>
       <item>
        <widget class="QLabel" name="Label_DOB">
         <property name="font">
          <font>
           <family>Lato</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Date of Birth:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="DateEdit_DOB">
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>11</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>180</y>
        <width>451</width>
        <height>201</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="ReferralReason">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="Label_Reason">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Reason for Referral</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LineEdit_Reason">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Example</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="assets/qrc/app-bg.qrc"/>
 </resources>
 <connections/>
</ui>
